code/script.js

```diff
@@ -30,18 +34,114 @@ const showMessage = (message, sender) => {
   chat.scrollTop = chat.scrollHeight
 }
 
-// Starts here
+// Starts here with a start button leading to the chat and inital greeting (with timer)
+
+startBtn.onclick = () => {
+  startPage.style.display = "none"
+  setTimeout(greeting, 1200)
+}
+
 const greeting = () => {
-  showMessage(`Hello there, What's your name?`, 'bot')
-  // Just to check it out, change 'bot' to 'user' here 👆
+  showMessage(`Hi there, who is this?`, 'bot') // bot greeting here
+}
+
+const handleNameInput = (event) => {
+  event.preventDefault();
+  const name = nameInput.value;
+  showMessage(`It's me, ${name}. Please help!`, 'user');
+  nameInput.value = "";
+  setTimeout(() => showDecade(name), 1000);  // users first answer, when submitting answer peage refresh is prevented by (event)
 }
 
-// Set up your eventlisteners here
+const showDecade = (name) => { // follow upp question by bot, showing first options buttons
+  showMessage(`Welcome ${name}. Of course! First, please tell me what decade you feel like diving into?`, 'bot');
+  setTimeout(() => selectMovie(year), 1000); 
```

Comment:
I think this line of code should be deleted, `selectMovie` is called when a button is clicked.